### PR TITLE
Fix saliency map norm to remove class cross-influence

### PR DESCRIPTION
**Pull request by Purg**

### CODE REVIEW

**Review by brianhhu — APPROVED** [review]
LGTM, thanks!

File: examples/.gitignore [hunk 1]
```diff
@@ -1,2 +1,2 @@
 **/imagenet_classes.txt*
-
+/*.jpg
```
Comment (vbhavank) [inline examples/.gitignore:2]: No comment, but I am curious about what this was trying to achieve, given my experience with git is limited?

**Review by vbhavank — APPROVED** [review]

File: examples/.gitignore (hunk 1 above)
Comment (brianhhu) [reply examples/.gitignore:2]: This will ignore the downloaded *.jpg files in the git repo such that they don't show up as untracked files.

Comment (Purg) [reply examples/.gitignore:2]: This is a specific sub-directory scope git ignore file, so this means "ignore any files matching the glob '*.jpg' in this directory."

Comment (vbhavank) [reply examples/.gitignore:2]: Got it. Thank you so much for the clarifications. I assumed it was something related to catdog.jpg surgery. I was interested in learning how that fix would look.

**Pull request merged**